R/dettl.R

```diff
@@ -42,7 +44,25 @@ DataImport <- R6::R6Class(
     require_branch = NULL,
     db_name = NULL,
     mode = NULL,
-    transaction = TRUE
+    load_in_transaction = TRUE,
+
+    #' @description
+    #' Tidy up open connections, rolling back any active transactions
+    close_connection = function() {
+      if (private$transaction_active) {
+        warning("Rolling back active transaction")
```

Comment:
I am not sold on warning here - warnings get grouped together and are displayed at some future point. If you *really* want a warning, make it immediate, but TBH this looks more like a message (we're telling the user we're doing something) than anything else (I generally just avoid warnings)

Reply:
Yeah that is a message really - have updated that